### DISCUSSION TITLE
Export as Markdown

### DISCUSSION THREAD

**Discussion by rutschi100** [massCodeIO/massCode#187] [Ideas]
I often have to develop a certain knowhow and document it. I like to use MassCode to write down the commands that are needed.

It would be really helpful if the snippets that are in a folder could be exported as Markdown. That way you could easily publish a cheat sheet or similar.

**Comment by antonreshetov** [comment]
Thanks for the suggestions, I will think about the implementation.

**Comment by a4ex** [comment]
I believe this is something that is really missing from app. Obviously there are ways to exported any required snippets from the "db.json" but, it would be very helpful to have an export option to .md file.

**Comment by pa-0** [comment]
FYI https://github.com/tomasmark79/masscode2md is a semi-functional workaround to this.  I have tried it and it works OK. The only current issue is that it does not handle fragments (only exporting the last fragment in a snippet to markdown).  I have reported this issue to the creator, and am trying to resolve it myself as well.  If this gets fixed, I think this will be an acceptable method of export.

**Reply by tomasmark79** [reply]
Updated repo here: With fragments export included.
https://github.com/tomasmark79/MassCode2MdFree

**Comment by wrufeger** [comment]
If someone still needs a markdown v2 export from the sqlite database, here is my little python script:
https://gist.github.com/wrufeger/84bd618bb0293c4abdc6363dddfa3775
